src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -75,7 +76,10 @@ export const ConversationRoute: React.FC<ConversationRouteProps> = props => {
               selectedConversationID={me.conversation.internalID}
             />
           </Media>
-          <Conversation conversation={me.conversation} />
+          <Conversation
+            conversation={me.conversation}
+            relayRefetch={props.relay}
```

Comment:
Maybe just name this prop `relay` so that things are 1:1?

Reply:
After looking into it, this can't be renamed to `relay`. `Conversation` here is a fragment container which injects its own `relay` prop into the child component where this would need to be accessed.

Reply:
Maybe just pass `this.relay.refetch` then? It seems a bit misleading to pass a relay instance but name it something else.